index.html

```diff
@@ -0,0 +1,153 @@
+            <h3 class="welcome center">"Hello! Welcome to the party next the river"</h3>
+            <h2 class="title">QUEVEDO FEST 2023</h2>
+            <p class="main-description back-desp"> Quevedo City electronic music festival features top DJs and electronic artists. Expect a mix of techno, house, trance, and more. Multiple stages, art installations, food, drinks, and activities. A must-attend event for electronic music fans. Get ready to dance and immerse in the electrifying atmosphere. </p>
+            <p class="main-description contact-about">Please contact us per Email for further questions <br class="desktop-ver"> about Quevedo fest 2023 <br> <a href="quevedo_fest2023@gmail.com"> quevedo_fest2023@gmail.com</a></p>
+        </div>
+        
+        <div class="main-about">
+            <h2 class="title-logo">Quevedo Fest 2023 Logo</h2>
+            <hr class="divider-title">
+            <p class="description-logo"> The Quevedo City Fest logo features vibrant colors and a carnival-inspired design, perfectly capturing the energy and excitement of the electronic music festival. With the river as its backdrop, the logo embodies the festival's unique setting, where nature and music come together to create a truly unforgettable experience.</p>
+            <img class="logo-image" src="images/logofestivalbg.png" alt=" festival logo">
+        </div>
+        <hr class="hr-section">
+        <div class="past-events">
+            <h2 class="title-logo past-fests">See the past Quevedo fest shows</h2>
+            <hr class="divider-title">
+            <p class="description-logo">Take a look at the last two festivals en Quevedo</p>
+            <div class="row-images">
+                <img class="concert-image first" src="images/concert1edit.jpg" alt="Concert 2021">
+                <img class="concert-image second" src="images/concert222.jpg" alt="Concert 2022">
+            </div>
+        </div>
+    </section>
+
+    
+    <!-- <hr class="hr-section"> -->
+
+    <section id="partners" class="partners">
+        <h2 class="partner-title">Partner</h2>
+        <hr class="divider-title">
+        <div class="partner">
+            <img class="sponsor-img" src="images/aldialogo.png" alt="">
+            <img class="sponsor-img" src="images/importadoracastro.png" alt="">
+            <img class="sponsor-img" src="images/laorientall.png" alt="">
+            <img class="sponsor-img" src="images/municipiol.png" alt="">
+            <img class="sponsor-img" src="images/Logotipo-Alfanet.png" alt="">
+        </div>
+        
+    </section>
+    <footer id="footer" class="footer">
+        
+        <img class="logo-footer" src="images/logofestivalbg.png" alt=" festival logo">
+        <div class="text-footer">
+        <h3 class="title-footer">2023 Alternative Events. <br> Some rigths reserved</h3>
+        <h4 class="description-footer">Quevedo fest is a registered branch of Alternative Events</h4>
+        </div>
+        
```

Comment:
- [ ] Hi. As part of the requirement you are supposed to follow the provided design as much as possible so please try considering the below change suggestions
- The partner section should only be present on the home page not on the about page
- The footer on the about page is black. Please consider changing the color

**Provided design**
<img width="862" alt="image" src="https://user-images.githubusercontent.com/37901469/217676950-dfc3e20a-b24a-4538-9f1f-91948fc54670.png">


**your UI**
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/37901469/217677036-ed9aad0a-9335-4704-bfa8-a61822a0fa91.png">